assets/javascripts/discourse/components/chat-composer.js

```diff
@@ -32,6 +32,11 @@ export function addChatToolbarButton(toolbarButton) {
   toolbarButtons.push(toolbarButton);
 }
 
+const _extraAutocompletes = [];
+export function addChatAutocompleteFn(fn) {
+  _extraAutocompletes.push(fn);
```

Comment:
Whenever adding a module local API like this we also need a `reset` method that is called after each test run.